<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lrc Cleaner</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTextBrowser" name="originText">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QTextEdit" name="editedText">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>396</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QPushButton" name="openButton">
       <property name="text">
        <string>打开</string>
       </property>
      </widget>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="halfBracketCheck">
       <property name="text">
        <string>( ) </string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="squareBracketCheck">
       <property name="text">
        <string>[ ]</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="bigBracketCheck">
       <property name="text">
        <string>{ }</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="angelBracketCheck">
       <property name="text">
        <string>&lt; &gt;</string>
       </property>
      </widget>
      <widget class="QLCDNumber" name="lcdNumber">
       <property name="font">
        <font>
         <family>ADMUI3Lg</family>
         <pointsize>14</pointsize>
        </font>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
